.github/ISSUE_TEMPLATE/3_product_release_checks.md

```diff
@@ -2,50 +2,39 @@
 name: Product Release Checks
 about: Use this template to track all release checks for your component
 title: "[Product] Release Checks"
-labels: compliance, documentation, "security analysis", "test results"
+labels: foss
```

Comment:
Is the `foss` label necessary. Tractus-X is an open-source project, so I don't see what else we should have here.
What specific aspect should the label highlight?

Reply:
We introduced the foss label because in the [Release Overview page](https://github.com/orgs/eclipse-tractusx/projects/26/views/17?filterQuery=issue-type%3Arelease_ac+is%3Aopen+) we differentiate between Issues between FOSS and KITs.

Reply:
Ah ok I see. Then I would suggest to still remove the `foss` label. If you do not want to show Kit content, then just use the `-label: kit` filter. `foss` is misleading to me, since KITs are also FOSS

Reply:
agree; ok for me as it is still possible to filter the two categories

Reply:
done